lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -40,9 +40,27 @@ if test "x$suse_cloud_version" != "x5"; then
 fi
 
 
+### Restart some services
+echo_summary "Restarting required services..."
+
+if test ! -f /etc/chef/amqp_passwd; then
+  grep -m1 "^amqp_pass " /etc/chef/server.rb | sed 's/amqp_pass "\([^"]*\)"/\1/g' > /etc/chef/amqp_passwd
```

Comment:
And that's because that file used to be inside rubygem-chef-server and moved to rubygem-chef-server-api. Since there's no rubygem-chef-server package anymore, the file is back up to a .rpmsave file.